### ISSUE TITLE
Documentation for embedding configuration shows wrong WrenErrorFn signature

### ISSUE THREAD

**Issue by asasine** [munificent/wren#529]
The signature for `WrenErrorFn` is:
```c
void error(
      WrenVM* vm,
      WrenErrorType type,
      const char* module,
      int line,
      const char* message)
```
However, the documentation [here](http://wren.io/embedding/configuring-the-vm.html#errorfn) shows it without the `WrenVM* vm` pointer as:
```c
void error( 
      WrenErrorType type, 
      const char* module, 
      int line, 
      const char* message)
```

**Comment by bjorn** [comment]
Already fixed by #509, but website needs to be rebuilt (I hope #521 will make it).

**Comment by asasine** [comment]
Wow I searched the issues but I guess I missed that one. I'll close this but likewise I am rooting for #521
